app/scripts/utils/antimeridian.test.ts

```diff
@@ -0,0 +1,243 @@
+import { Feature, MultiPolygon, Polygon } from 'geojson';
+import { fixAntimeridian } from './antimeridian';
+
+describe('Antimeridian', () => {
+  it('Feature in 1st earth east', () => {
+    const feature: Feature<Polygon> = {
+      type: 'Feature',
+      properties: {},
+      geometry: {
+        coordinates: [
+          [
+            [190, 20],
+            [190, -20],
+            [200, -20],
+            [200, 20],
+            [190, 20]
+          ]
+        ],
+        type: 'Polygon'
+      }
+    };
+
+    const expected: Feature<MultiPolygon> = {
+      type: 'Feature',
+      properties: {},
+      geometry: {
+        coordinates: [
+          [
+            [
+              [-170, -20],
```

Comment:
Can you point me where the order of the coordinates changes? I can't wrap my head around it.

Reply:
That was also strange to me, but for some reason the turf.js function (intersect and difference), reorder the coordinates when cutting the polygons.